src/core.js

```diff
@@ -395,7 +395,7 @@ jQuery.extend( {
 		}
 
 		// Flatten any nested arrays
-		return concat.apply( [], ret );
+		return flat.call( ret );
```

Comment:
I think it might be smaller if the function just accepted a parameter: `flat( ret )`, without using `call`. You may try such an implementation & see if running `grunt` shows a smaller size increase.

Reply:
Done